docs/tutorial/1-serialization.md

```diff
@@ -124,59 +127,59 @@ We can actually also save ourselves some time by using the `ModelSerializer` cla
 ## Working with Serializers
 
 Before we go any further we'll familiarize ourselves with using our new Serializer class.  Let's drop into the Django shell.
-
-    python manage.py shell
-
+```bash
+python manage.py shell
+```
 Okay, once we've got a few imports out of the way, let's create a couple of code snippets to work with.
-
-    from snippets.models import Snippet
-    from snippets.serializers import SnippetSerializer
-    from rest_framework.renderers import JSONRenderer
-    from rest_framework.parsers import JSONParser
-
-    snippet = Snippet(code='foo = "bar"\n')
-    snippet.save()
-
-    snippet = Snippet(code='print("hello, world")\n')
-    snippet.save()
-
+```python
+from snippets.models import Snippet
+from snippets.serializers import SnippetSerializer
+from rest_framework.renderers import JSONRenderer
+from rest_framework.parsers import JSONParser
+
+snippet = Snippet(code='foo = "bar"\n')
+snippet.save()
+
+snippet = Snippet(code='print("hello, world")\n')
+snippet.save()
+```
 We've now got a few snippet instances to play with.  Let's take a look at serializing one of those instances.
-
-    serializer = SnippetSerializer(snippet)
-    serializer.data
-    # {'id': 2, 'title': '', 'code': 'print("hello, world")\n', 'linenos': False, 'language': 'python', 'style': 'friendly'}
-
+```python
+serializer = SnippetSerializer(snippet)
+serializer.data
+# {'id': 2, 'title': '', 'code': 'print("hello, world")\n', 'linenos': False, 'language': 'python', 'style': 'friendly'}
+```
```

Comment:
For these examples which are in the Python REPL, there is a dedicated language for it `pycon`, which means you can put `>>>` in front of the prompts, and we don't need to comment the output:

	```pycon
	>>> serializer = SnippetSerializer(snippet)
	>>> serializer.data
	{'id': 2, 'title': '', 'code': 'print("hello, world")\n', 'linenos': False, 'language': 'python', 'style': 'friendly'}
	```

Syntax highlighting should work in both GitHub and the docs from what I can tell.